tests/countries/test_indonesia.py

```diff
@@ -343,8 +344,8 @@ def test_whit_monday(self):
     def test_pancasila_day(self):
         name = "Hari Lahir Pancasila"
         self.assertHolidayName(name, (f"{year}-06-01" for year in range(2016, 2050)))
-        self.assertNoHoliday(
```

Comment:
But it's the same here: if we check that this name is not present at all in these years (L350), then why check whether it is not present on certain dates of same years?

Reply:
Right. It's the release day and I'd like to include this one. Let's take another look at it later -- it's a lower priority test code.